.github/disabled-workflows/mac_test.yml

```diff
@@ -0,0 +1,95 @@
+name: Mac Testing
```

Comment:
Any workflow in the disabled-workflows folder should be considered sorta-working but turned off, most likely because we are waiting until we do self-hosted for more horsepower.